### DISCUSSION TITLE
Exclude string with multiple words

### DISCUSSION THREAD

**Discussion by huguestennier** [sapegin/textlint-rule-terminology#66] [Ideas]
Maybe I'm doing this wrong, but is there a way to exclude a whole string?

For example, I want to replace `Front-end` with => `frontend`. That works

But I want to exclude `New Front-End Adventures` because that's how the author titled the workshop. 

Is there a way to do that? I tried both `skip` and `exclude`

**Comment by sapegin** [comment]
`exclude` exclude terms from the [built-in dictionary](https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.json), and `skip` skips TextLint node types, like blockquotes, etc.

The only way to somehow achieve what you're trying to do is to disable the rule for the entire paragraph:

https://github.com/sapegin/textlint-rule-terminology#tips--tricks

**Comment by sapegin** [comment]
Having a special feature for that would be nice though — feel free to send a pull request 👾

**Comment by mghill** [comment]
We are using https://github.com/textlint/textlint-filter-rule-allowlist for this purpose, FYI as it lets you omit certain words or phrases from all textlint rules
